internal/controller/nodemaintenance_controller.go

```diff
@@ -50,9 +50,10 @@ import (
 )
 
 var (
-	defaultMaxNodeMaintenanceTime = 1600 * time.Second
-	waitPodCompletionRequeueTime  = 10 * time.Second
-	drainReqeueTime               = 10 * time.Second
+	defaultMaxNodeMaintenanceTime   = 1600 * time.Second
```

Comment:
nit: Time > Seconds? 
It is ok for me keep the "time" postfix if you prefer so

Reply:
lets keeps as is :)